sn_networking/src/lib.rs

```diff
@@ -621,6 +622,43 @@ impl Network {
             .await)
     }
 
+    pub async fn get_store_cost_from_network(&self, dbc_id_key: NetworkAddress) -> Result<Token> {
```

Comment:
this shall be a more generic record_entry, i.e. the target could be a chunk, register or DBC itself ?